File: Java/JavaDatabaseFrameworks/Common/src/main/java/org/dbFrameworks/TestsResults.java
package org.dbFrameworks;

import java.util.ArrayList;
import java.util.List;

public class TestsResults {
    private List<TestResult> _testResults = new ArrayList<TestResult>();
    public void Add(TestResult testResult) { _testResults.add(testResult); }

    @Override
    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append("{\n"+
                "   \"Language\": Java\n"+
                "   \"Results\": [\n");
        for(TestResult testResult : _testResults){
            sb.append("          "+testResult.toString()+"\n");
        }
        sb.append("   ]'\n}");
        return sb.toString();
    }
}

File: Python/Django ORM/Common/TestsResults.py
class TestResults:
    def __init__(self):
        self.tests = []
    def Add(self, test_result):
        self.tests.append (test_result)

    def __repr__(self):
        result ="""
"Language": "Python"
"Results": ["""
        for test in self.tests:
            result += str(test) + "\n"
        result+="     ]\n}"
        return result

